packages/performance/src/utils/attributes_utils.ts

```diff
@@ -88,3 +93,17 @@ export function getEffectiveConnectionType(): EffectiveConnectionType {
       return EffectiveConnectionType.UNKNOWN;
   }
 }
+
+export function isValidCustomAttributeName(name: string): boolean {
+  if (name.length === 0 || name.length > MAX_ATTRIBUTE_NAME_LENGTH) {
+    return false;
+  }
+  const matchesReservedPrefix = RESERVED_ATTRIBUTE_PREFIXES.some(prefix =>
```

Comment:
`Array.some` is an ES6 feature. Please add `import 'core-js/features/array/some';` in https://github.com/firebase/firebase-js-sdk/blob/master/packages/polyfill/index.ts

Though we don't ship polyfills to npm packages, `firebase.js` still relies on it.

Reply:
Thank you, done